src/platforms/common/enriching-events/user-feedback.mdx

```diff
@@ -53,7 +54,7 @@ SentrySdk.CaptureUserFeedback(eventId, "user@example.com", "It broke.", "The Use
 
 </PlatformSection>
 
-<PlatformSection notSupported={["android", "apple", "java", "native", "dart", "flutter", "ruby", "unity"]}>
+<PlatformSection notSupported={["android", "apple", "java", "native", "dart", "flutter", "ruby", "unity", "react-native"]}>
```

Comment:
Once we add the manual API includes, we can simply remove the top `notSupported`.